src/shx.js

```diff
@@ -5,7 +5,18 @@ import help from './help';
 import { CMD_BLACKLIST, EXIT_CODES, CONFIG_FILE } from './config';
 import { printCmdRet } from './printCmdRet';
 import path from 'path';
-import pathExists from 'path-exists';
+import fs from 'fs';
+import objAssign from 'es6-object-assign';
+objAssign.polyfill(); // modifies the global object
+
+const pathExistsSync = (filePath) => {
```

Comment:
I know it's a tiny amount of code, but Sindre did write [`path-exists`](https://www.npmjs.com/package/path-exists) for this.  If nothing else, it is one less utility for us to manage.